elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -434,16 +437,16 @@ class RhTab extends PFElement {
   }
 
   constructor() {
-    super(RhTab);
-
-    if (!this.id) {
-      this.id = `${RhTab.tag}-${generateId()}`;
-    }
+    super(PfeTab);
   }
 
   connectedCallback() {
     super.connectedCallback();
 
+    if (!this.id) {
```

Comment:
Can we update this to attach a `pfe-id` instead of using ids?  This causes some issues for us when building A/B tests with the Adobe Test & Target tool because it will always see and try to use the ID (it's a GUI) and fail because this is randomly generated.

Reply:
@castastrophe It won't be randomly generated if the id attribute is already present on the element. Is that still a problem if you give it an ID ahead of time?

Reply:
That means every time we use the pattern we'd have to include instructions that it has to have an ID on it.  Is the ID important to the connection?  What would it hurt to use a more internal way of connecting the parts?